app/policies/defence_request_policy.rb

```diff
@@ -8,11 +8,11 @@ def initialize(user, scope)
     end
 
     def resolve
-      if user.cso?
+      if user.roles.include?("cso")
         scope.not_aborted
-      elsif user.cco?
+      elsif user.roles.include?("cco")
```

Comment:
Just wondering if there was a reason you didn't use the cco method from line 103 below? And similarly for cso etc.